app/layouts/default.vue

```diff
@@ -1,12 +1,14 @@
 <template>
-  <div>
+  <div class="flex flex-col min-h-screen">
     <Navbar />
-    <main class="p-5">
+    <main class="flex-grow p-5">
```

Comment:
I was thinking the main can take up the entire width of the viewport on small screens, and we can add some margin on larger screens?

Reply:
I agree with this as well